LICENSE

```diff
@@ -198,4 +198,4 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
```

Comment:
Is it possible to exclude this file from linting? I'm afraid some of the GitHub infra may consider this a custom license even if we add as much as a newline.

Reply:
1) It is possible to exclude from linting
2) You needn't be afraid of that, this edit was introduced by the "end of file fixer" which I also use in other projects (e.g. [fugit](https://github.com/lmmx/fugit/blob/master/LICENSE.txt) which do not invalidate upon linting.

Licenses aren't detected in a fragile way (like a MD5 checksum/exact match), we can tell that from e.g. MIT licensed projects where the license is detected despite the user editing it (you have to put your name in it for it to be valid).

Specifically the docs on [how licenses are detected on GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#detecting-a-license) using the open source Ruby gem [licensee](https://github.com/licensee/licensee) using several "matchers" which notes that its method includes

> If we strip away whitespace and copyright notice,

Which confirms that the whitespace change is not going to obscure the license [it does in fact do more than that using string matching but as I say that wouldn't become relevant]